microsoft-edge/webview2/concepts/versioning.md

```diff
@@ -27,29 +24,44 @@ The prerelease package is a superset of the release package with the additional
 *   .NET APIs: [WPF][DotnetMicrosoftWebWebview2WpfNamespace], [WinForms][DotnetMicrosoftWebWebview2WinformsNamespace], and [Core][DotnetMicrosoftWebWebview2CoreNamespace]  
 *   Experimental APIs:  For more information, navigate to the [Experimental APIs](#experimental-apis) section.  
 
-## Experimental APIs  
+### Experimental APIs  
 
 The WebView team is testing experimental APIs that may be included in future releases.  The experimental APIs are marked as `experimental` in the SDK.  Experimental APIs may ship as fully stable APIs in the release package.  You can evaluate the Experimental APIs and share feedback using the [WebView feedback repo][GithubMicrosoftedgeWebviewfeedback].  
 
+
+
 > [!CAUTION]
-> Avoid using the experimental APIs in production apps.  
+> Experimental APIs may be introduced, modified, and removed from SDK to SDK.  Experimental APIs may not be available in your installed version of the WebView2 Runtime.  Avoid using the experimental APIs in production apps.  
+
+### Roadmap  
+
+The release package contains all of the stable, supported Win32 C/C++ APIs.  In the future, the release package will contain all stable, supported .NET APIs when they are made generally available.  The prerelease package contains experimental APIs that are subject to change based upon your feedback and shared insights. 
 
 ## Matching WebView2 Runtime versions  
+To develop a WebView2 application, you must install either the [WebView2 Runtime][MicrosoftDeveloperEdgeWebview2] or a [non-stable Microsoft Edge channel][MicrosoftedgeinsiderDownload]. There are two things to take into account when determining the minimum version of the [WebView2 Runtime][MicrosoftDeveloperEdgeWebview2] required.
```

Comment:
It's helpful but I'm not sure the main task here is to help developers determine min RT version since developers don't control RT version anyway. I think we should focus on what we want developers to understand and do, and cover these points,

* How does SDK, RT, and browser version scheme work and how they match up with each other. 
* Each time we release a Release SDK, a RT update with corresponding or higher version is also released to the end users. Generally speaking you can use all the APIs in the SDK, but as a best practice you should QI/try-catch on the newer APIs as some users may be disconnected from RT updates (e.g. no internet and enterprise controlling RT version). 
* Similarly, each time we release a Pre-release SDK, a Canary update with corresponding or higher version is also released to the end users. Canary is ahead of all other channels (Dev/beta/RT), so some API in the SDK may not work on other channels. 
* Each SDK has a min version, but it's the min version where some APIs, not all APIs in the SDK, can run. Explain how you can find out API's min RT version if developer needs it.